assets/help/cs/help.html

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html><head><meta charset="UTF-8" /><meta name="viewport" content="initial-scale=1.0" /><link rel="stylesheet" href="../help.css" /></head><body><ul id="top">
+<html><head><meta charset="UTF-8" /><meta name="color-scheme" content="dark light"><meta name="viewport" content="initial-scale=1.0" /><link rel="stylesheet" href="../help.css" /></head><body><ul id="top">
```

Comment:
why are the changes in help.html needed? does it make a difference? for me, also without this PR, i have dark mode on the help pages, even without flickering on opening.

Reply:
Hmmm yeah I checked just now, it really is already dark. I guess it's because of [this call to `setForceDark`](https://github.com/deltachat/deltachat-android/blob/920889c1193e826bc2851f2e5a45e6400b439b32/src/org/thoughtcrime/securesms/LocalHelpActivity.java#L16)?
But [the docs say that `setForceDark` is deprecated](https://developer.android.com/reference/android/webkit/WebSettings#setForceDark(int)), so I think this is a better way to go about this. Well, except for the part where I change the generated files.

Reply:
I moved it to CSS.

Reply:
Plus removed the `setForceDark` (again, I haven't tested it).